pkgs/development/python-modules/ipykernel/tests.nix

```diff
@@ -23,7 +25,10 @@ buildPythonPackage {
     ipykernel
     ipyparallel
     nose
```

Comment:
```suggestion
```
nose is an obsolete package and we don't need it anymore.
```
ipykernel/tests/utils.py:154: in new_kernel
    import nose
/nix/store/gswp9km1wzj1lrlwcjp2qvixzqicf6xr-python3.10-nose-1.3.7/lib/python3.10/site-packages/nose/__init__.py:1: in <module>
    from nose.core import collector, main, run, run_exit, runmodule
/nix/store/gswp9km1wzj1lrlwcjp2qvixzqicf6xr-python3.10-nose-1.3.7/lib/python3.10/site-packages/nose/core.py:11: in <module>
    from nose.config import Config, all_config_files
/nix/store/gswp9km1wzj1lrlwcjp2qvixzqicf6xr-python3.10-nose-1.3.7/lib/python3.10/site-packages/nose/config.py:9: in <module>
    from nose.plugins.manager import NoPlugins
/nix/store/gswp9km1wzj1lrlwcjp2qvixzqicf6xr-python3.10-nose-1.3.7/lib/python3.10/site-packages/nose/plugins/__init__.py:185: in <module>
    from nose.plugins.manager import *
/nix/store/gswp9km1wzj1lrlwcjp2qvixzqicf6xr-python3.10-nose-1.3.7/lib/python3.10/site-packages/nose/plugins/manager.py:418: in <module>
    import pkg_resources
```

Reply:
Nice ! I removed it :)

Still a few errors though:
```
ERROR ipykernel/tests/test_eventloop.py::test_asyncio_interrupt - pytest.PytestUnhandledThreadExceptionWarning: Exception in thread Thread-8 ...
FAILED ipykernel/tests/test_embed_kernel.py::test_embed_kernel_func - DeprecationWarning: Passing unrecognized arguments to super(IPythonKernel)....
FAILED ipykernel/tests/test_kernelspec.py::test_install_kernelspec - traitlets.config.configurable.MultipleInstanceError: An incompatible siblin...
```

Reply:
It seems to be broken on the master as well.
To be honest, I don't know what the cause is.

Reply:
Those failures are weird because the tests run fine for the same version on a classic (venv based) setup.

Reply:
Maybe we should just ignore those tests...